### ISSUE TITLE
Missing quality options

### ISSUE THREAD

**Issue by dimqua** [iv-org/invidious#656]
I noticed that if your default quality setting is `hd720`, it's not possible to set higher quality (such as `dash`) for each particular video manually.
![](https://user-images.githubusercontent.com/540898/62227452-a08bf700-b3c4-11e9-867a-4a3024d2e858.png)
If you have `dash` as default then you can manually switch to `1080p` quality, but what's the point? Why isn't it allowed to set 1080p as default quality then?
![invidious](https://user-images.githubusercontent.com/540898/62227639-f5c80880-b3c4-11e9-874e-abfd19207e9d.png)

**Comment by TheFrenchGhosty** [comment]
From what I know YouTube only provide 720p (hd720 mp4) and 360p (medium) with audio + video, having something else require to mux (merge) two different sources (the audio and the video) together, for that the server need to "work" making it then impossible to stream directly from google server.

TL;DR : It's a YouTube "feature"


@cloudrac3r did some research about it, so I'm sure she will have more details.

**Comment by omarroth** [comment]
Technically, calling `dash` a quality is misleading, since it provides multiple resolutions in a different format (including 720p). There's some more info in #548, but it might make sense to call it a "format" instead, similar to what's mentioned in #97.

Important to note is that DASH requires it's own quality switcher, which is why the hd720/medium qualities and dash qualities have a different menu.

See also: #646.

**Comment by shakeyourbunny** [comment]
It is really annoying to manually set the video quality to 1080p... and my installed instance defaults to 480p although I set it to DASH.
